st2common/st2common/services/policies.py

```diff
@@ -0,0 +1,69 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+from st2common.constants import action as ac_const
```

Comment:
I noticed you started using short hand variable names for various imports in your recent changes (ac_ex_db -> execution_db, etc.).

Would be good to use consistent name across the code base, since in most of the old / existing code base we use different aliases names for the imports and variables.

Reply:
My plan is to start making similar changes across the code base. :P

Reply:
Noooo :P

Reply:
Seriously though, i plan to clean up some existing code base.

Reply:
I'm not a fan of it - there is really no need, unless the code is not using good variable names (which might be the case in some of the old existing code).

I don't like the short hand notation, especially since 90% of the existing code already uses variable names such as `execution_db`, `execution_api`, etc.

Reply:
I can probably live with execution_db, etc as is. But the individual import of action constants are just unsightly not to mention unnecessary as are other module import styles.